jwst/master_background/expand_to_2d.py

```diff
@@ -0,0 +1,314 @@
+import math
+
+import numpy as np
+
+from .. import datamodels
+from .. assign_wcs import nirspec               # for NIRSpec IFU data
+
+def expand_to_2d(input, m_bkg_spec):
+    """Expand a 1-D background to 2-D.
+
+    Parameters
+    ----------
+    input : JWST data model
+        The input science data.
+
+    m_bkg_spec : string or JWST data model
+        Either the name of a file containing a 1-D background spectrum,
+        or a data model containing such a spectrum.
+
+    Returns
+    -------
+    background : JWST data model
+        A copy of `input` but with the data replaced by the background,
+        "expanded" from 1-D to 2-D.
+    """
+
+    with datamodels.open(m_bkg_spec) as bkg:
+        if len(bkg.spec) > 1:
+            print("Warning:  The input 1-D spectrum contains multiple spectra")
```

Comment:
Should use log.warning() instead of print, at least once this becomes part of the "official" step package.

Reply:
Yes, I forgot to change these.